tests/TestEnvironment.php

```diff
@@ -5,26 +5,18 @@
 namespace WMDE\Fundraising\MembershipContext\Tests;
 
 use Doctrine\ORM\EntityManager;
+use Exception;
 
 class TestEnvironment {
 
 	private TestMembershipContextFactory $factory;
 
-	private function __construct( array $config ) {
-		$this->factory = new TestMembershipContextFactory( $config );
+	private function __construct() {
+		$this->factory = new TestMembershipContextFactory();
 	}
 
 	public static function newInstance(): self {
-		$environment = new self(
```

Comment:
See my comment for `TestMembershipContextFactory.php`. We can drop the "outer" array, but should keep the database configuration array

Reply:
Done.